i18n/src/commonMain/moko-resources/base/strings.xml

```diff
@@ -190,6 +190,10 @@
     <string name="onboarding_permission_notifications_description">Get notified for library updates and more.</string>
     <string name="onboarding_permission_ignore_battery_opts">Background battery usage</string>
     <string name="onboarding_permission_ignore_battery_opts_description">Avoid interruptions to long-running library updates, downloads, and backup restores.</string>
+    <string name="onboarding_permission_crashlytics">Send crash logs</string>
+    <string name="onboarding_permission_crashlytics_description">Send anonymised crash logs to the developers.</string>
```

Comment:
You used the British spelling here and the American spelling in the analytics string. The app uses American English in the default strings (e.g. `color` instead of `colour`).
```suggestion
    <string name="onboarding_permission_crashlytics_description">Send anonymized crash logs to the developers.</string>
```